### PR TITLE
Update Ansible/Vagrant process

### PR DESCRIPTION
**Pull request by ronichoudhury**
This makes Vagrant/Ansible provisioning work, providing some options to the provisioning process for setting certain values in the deployment, such as the admin user password, and which git revisions to use for dependent software, etc.

### CODE REVIEW

File: app/resonant-laboratory/ansible/xenial/girder.service.template [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Girder
+After=network.target
+
+[Service]
```
Comment (danlamanna) [inline app/resonant-laboratory/ansible/xenial/girder.service.template]: At a minimum I would put a User/Group here so Girder doesn't run as root. Example [here](https://github.com/girder/girder/blob/master/devops/ansible/roles/girder/templates/daemon/girder.service.j2#L6-L7). Same goes for the upstart config.

File: app/resonant-laboratory/ansible/library/girder.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python
```
Comment (danlamanna) [inline app/resonant-laboratory/ansible/library/girder.py]: If you wanted, you could use the galaxy role for no other purpose than to bring this module in automatically.

File: app/resonant-laboratory/ansible/xenial/girder.service.template (hunk 1 above)
Comment (ronichoudhury) [inline app/resonant-laboratory/ansible/xenial/girder.service.template]: I'll need to confer with IT before I make this change, so here's an issue describing the problem: #390

File: app/resonant-laboratory/ansible/library/girder.py (hunk 2 above)
Comment (ronichoudhury) [inline app/resonant-laboratory/ansible/library/girder.py]: Cool. I'll be in touch about this.

**Pull request merged**